### PR TITLE
normalize test fix

### PR DESCRIPTION
**Pull request by k223kim**
<details>
  <summary><b>Before submitting</b></summary>

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## What does this PR do?

Normalize PR( #530 ) had issues with the test. Fixed this by avoiding input tensors to be close to 0. (Thanks Tom for the suggestion!)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
⚡️

### CODE REVIEW

File: thunder/tests/opinfos.py [hunk 1]
```diff
@@ -5931,11 +5931,13 @@ def normalize_sample_generator(op, device, dtype, requires_grad, **kwargs):
         (4, 2, 4, 5),
     )
     for case in cases:
-        yield SampleInput(make(case), eps=1e-8)
-        yield SampleInput(make(case), p=0, eps=1e-8)
-        yield SampleInput(make(case), p=1, eps=1e-8)
-        yield SampleInput(make(case), p=4, eps=1e-8)
-        yield SampleInput(make(case), p=math.inf, eps=1e-8)
+        input_tensor = make(case)
+        input_tensor = input_tensor + 0.2 * torch.sign(input_tensor)
```
Comment (t-vi) [inline thunder/tests/opinfos.py:5935]:
```suggestion
        # avoid very small norm tensors, which can be unstable to normalize
        input_tensor = input_tensor + 0.2 * torch.sign(input_tensor)
```

**Review by t-vi — APPROVED** [review]
Thank you @k223kim

File: thunder/tests/opinfos.py [hunk 2]
```diff
@@ -5931,11 +5931,14 @@ def normalize_sample_generator(op, device, dtype, requires_grad, **kwargs):
...
     for case in cases:
-        yield SampleInput(make(case), eps=1e-8)
-        yield SampleInput(make(case), p=0, eps=1e-8)
-        yield SampleInput(make(case), p=1, eps=1e-8)
-        yield SampleInput(make(case), p=4, eps=1e-8)
-        yield SampleInput(make(case), p=math.inf, eps=1e-8)
+        input_tensor = make(case)
+        # avoid very small norm tensors, which can be unstable to normalize
+        input_tensor = input_tensor + 0.2 * torch.sign(input_tensor)
+        yield SampleInput(input_tensor, eps=1e-8)
```
Comment (mruberry) [inline thunder/tests/opinfos.py:5937]: Sample inputs shouldn't reuse the same tensor (which is a subtle property to check for), in case they mutate the tensor. Generate a new tensor each time using the same technique.

Can this use 

https://github.com/k223kim/lightning-thunder/blob/71b4ac9f71a895ef7c5dfe47c23657e2bfb3efb6/thunder/tests/opinfos.py#L66

with zero as a "singularity"?
